### ISSUE TITLE
Collapsable Grid Columns and Rows

### ISSUE THREAD

**Issue by trodent83** [dotnet/wpf#157]
Hi,

Currently there is no way to set the visibility of a hole column or a row. (or atleast not without a Workaround)

This would be more then desirable.

**Comment by stevenbrix** [comment]
@trodent83 this would be awesome! Right now we're focusing on getting the rest of our infrastructure ready so that we can release the rest of the framework, so we won't be able to get to this at the moment. Are you imagining that we add a `Visibility` property to Column/RowDefinition?

**Comment by thomasclaudiushuber** [comment]
You can set the Height of a RowDefinition to 0 or the Width of a ColumnDefinition to 0, which is actually a collapse of the Row or Column. Of course there's no virtualization happening for the elements that get out of view. 

What would this issue be, an additional property to do that?

**Comment by trodent83** [comment]
@thomasclaudiushuber Yes it would be an additional property. About the Setting the Row and Column Height and Width to 0 is a Solution and in most cases this is done in the usual Workaround where use a CoerceValueCallback to set the width and hight depending on another property.

But this hole Thing is a not to performat hack, which is basicly only a Workaround and there is no solution for that if I want to only hide the row or collumn and not collapse it (okay this is not too often).
